packages/ceramic-common/src/utils/doctype-utils.ts

```diff
@@ -163,7 +163,8 @@ export class DoctypeUtils {
      * @param schema - Doctype schema
      */
     static isSchemaValid(schema: object): boolean {
-        return this.validator.validateSchema(schema)
+        this.validator.compile(schema) // throws an error on invalid schema
```

Comment:
If it throws an error maybe this could be wrapped in a `try { ... return true } catch { ... return false }` statement so the caller doesn't have to catch the error explicitly?

Reply:
The error thrown contains information about the errors. That's why its better to throw it.